hardhat.config.js/index.js

```diff
@@ -34,6 +34,7 @@ const config = {
     recursive: false,
     timeout: 0,
     slow: 5000,
+    jobs: 3,
```

Comment:
Probably a good idea to use an environment variable (as you mentioned). May as well default to `1`.